src/test/routes/safe/store/actions/utils.test.ts

```diff
@@ -0,0 +1,170 @@
+import { getLastTx, getNewTxNonce, shouldExecuteTransaction } from 'src/logic/safe/store/actions/utils'
+import { getMockedSafeInstance, getMockedTxServiceModel } from 'src/test/utils/safeHelper'
+import axios from 'axios'
+import { buildTxServiceUrl } from 'src/logic/safe/transactions'
+
+describe('shouldExecuteTransaction', () => {
+  it('It should return true if given a safe with a threshold > 1',  async () => {
```

Comment:
it should return true if there are two confirmations, otherwise, for a new transaction it should return false

Reply:
```
expect(result).toBe(false)
```

oh yeah the test is correct but description is not